node/cn/tracers/api_test.go

```diff
@@ -179,136 +181,136 @@ func (b *testBackend) StateAtTransaction(ctx context.Context, block *types.Block
+		b.AddTx(tx)
+	}))
+
+	testSuite := []struct {
+		blockNumber rpc.BlockNumber
+		call        klaytnapi.CallArgs
+		config      *TraceConfig
+		expectErr   error
+		expect      interface{}
+	}{
+		// Standard JSON trace upon the genesis, plain transfer.
+		{
+			blockNumber: rpc.BlockNumber(0),
+			call: klaytnapi.CallArgs{
+				From:  accounts[0].addr,
+				To:    &accounts[1].addr,
+				Value: (hexutil.Big)(*big.NewInt(1000)),
+			},
+			config:    nil,
+			expectErr: nil,
+			expect: &klaytnapi.ExecutionResult{
+				Gas:         params.TxGas,
+				Failed:      false,
+				ReturnValue: "",
+				StructLogs:  []klaytnapi.StructLogRes{},
+			},
+		},
+		// Standard JSON trace upon the head, plain transfer.
+		{
+			blockNumber: rpc.BlockNumber(genBlocks),
+			call: klaytnapi.CallArgs{
+				From:  accounts[0].addr,
+				To:    &accounts[1].addr,
+				Value: (hexutil.Big)(*big.NewInt(1000)),
+			},
+			config:    nil,
+			expectErr: nil,
+			expect: &klaytnapi.ExecutionResult{
+				Gas:         params.TxGas,
+				Failed:      false,
+				ReturnValue: "",
+				StructLogs:  []klaytnapi.StructLogRes{},
+			},
+		},
+		// Standard JSON trace upon the non-existent block, error expects
+		{
+			blockNumber: rpc.BlockNumber(genBlocks + 1),
+			call: klaytnapi.CallArgs{
+				From:  accounts[0].addr,
+				To:    &accounts[1].addr,
+				Value: (hexutil.Big)(*big.NewInt(1000)),
+			},
+			config:    nil,
+			expectErr: fmt.Errorf("the block does not exist (block number: %d)", genBlocks+1),
+			expect:    nil,
+		},
+		// Standard JSON trace upon the latest block
+		{
+			blockNumber: rpc.LatestBlockNumber,
+			call: klaytnapi.CallArgs{
+				From:  accounts[0].addr,
+				To:    &accounts[1].addr,
+				Value: (hexutil.Big)(*big.NewInt(1000)),
+			},
+			config:    nil,
+			expectErr: nil,
+			expect: &klaytnapi.ExecutionResult{
+				Gas:         params.TxGas,
+				Failed:      false,
+				ReturnValue: "",
+				StructLogs:  []klaytnapi.StructLogRes{},
+			},
+		},
+		// Standard JSON trace upon the pending block
+		{
+			blockNumber: rpc.PendingBlockNumber,
+			call: klaytnapi.CallArgs{
+				From:  accounts[0].addr,
+				To:    &accounts[1].addr,
+				Value: (hexutil.Big)(*big.NewInt(1000)),
+			},
+			config:    nil,
+			expectErr: nil,
+			expect: &klaytnapi.ExecutionResult{
+				Gas:         params.TxGas,
+				Failed:      false,
+				ReturnValue: "",
+				StructLogs:  []klaytnapi.StructLogRes{},
+			},
+		},
+	}
+	for _, testspec := range testSuite {
+		result, err := api.TraceCall(context.Background(), testspec.call, rpc.BlockNumberOrHash{BlockNumber: &testspec.blockNumber}, testspec.config)
+		if testspec.expectErr != nil {
+			if err == nil {
+				t.Errorf("Expect error %v, get nothing", testspec.expectErr)
+				continue
+			}
+			if !reflect.DeepEqual(err, testspec.expectErr) {
+				t.Errorf("Error mismatch, want %v, get %v", testspec.expectErr, err)
+			}
+		} else {
+			if err != nil {
+				t.Errorf("Expect no error, get %v", err)
+				continue
```

Comment:
```suggestion
```

Reply:
Changed to use assert.Equal instead. https://github.com/klaytn/klaytn/pull/1965/commits/5d82e3aa76c012f8165f6437d0c08f03ee654858